File: PizzaBackendServer/WebApi/BLL/Models/MakeOrderModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DAL.Data;

namespace BLL.Models
{
    public class MakeOrderModel
    {
        public int ClientId { get; set; }
        public string Address { get; set; }
        public string PhoneNumber { get; set; }
        public string ClientName { get; set; }
        public string PayingType { get; set; }
        public DateTime? PredictedTime { get; set; }
        public decimal? Change { get; set; }
        public int? UsedBonuses { get; set; }
        public int GivenBonuses { get; set; }
        public string? Comment { get; set; }


        public MakeOrderModel() { }

        public MakeOrderModel(int clientId,  string address, string phoneNumber, string clientName, string payingType, decimal? change, DateTime? predictedTime, int? usedBonuses, int givenBonuses, string? comment)
        {
            ClientId = clientId;
            Address = address;
            PhoneNumber = phoneNumber;
            ClientName = clientName;
            PayingType = payingType;
            Change = change;
            PredictedTime = predictedTime;
            UsedBonuses = usedBonuses;
            GivenBonuses = givenBonuses;
            Comment = comment;
        }
    }
}

File: PizzaMobileApp/pizzer_mobile/lib/models/make_order_model.dart
class MakeOrderModel {
  int? clientId;
  String? address;
  String? phoneNumber;
  String? clientName;
  String? payingType;
  DateTime? predictedTime;
  int? change;
  int? usedBonuses;
  int? givenBonuses;
  String? comment;

  MakeOrderModel(
      {this.clientId,
      this.address,
      this.phoneNumber,
      this.clientName,
      this.payingType,
      this.predictedTime,
      this.change,
      this.usedBonuses,
      this.givenBonuses,
      this.comment});
}
